Dockerfile

```diff
@@ -1,20 +1,17 @@
-FROM node:stretch-slim as build-deps
-
 LABEL maintainer="helge.dzierzon@brockmann-consult.de"
```

Comment:
It seems that it is not possible to have the LABELs first. You will get an error message.

Reply:
Should have tried myself first...